testcases/kernel/syscalls/send/send01.c

```diff
@@ -103,6 +103,7 @@ static struct test_case_t tdat[] = {
 	,
 #ifndef UCLINUX
 	/* Skip since uClinux does not implement memory protection */
+#if 0 // Enable when sgx-lkl github issue 772 is fixed. Actually maybe never due to PROT_NONE not being detectable
```

Comment:
What's the problem here? 772 isn't really an issue I could go look at to try and fix.